### PR TITLE
Loki grafana

### PR DESCRIPTION
**Pull request by youssefazrak**
**What this PR does / why we need it**:
Implementation of Loki: Loki needs to be added as a datasource in Grafana.
This modifies the currents helm chart for Grafana and creates the Loki datasource yaml file and let Grafana loads it on startup.

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: config/monitoring/grafana/values.yaml [hunk 1]
```diff
@@ -59,6 +59,8 @@ grafana:
       - prometheus
       # - prometheus-thanos-query:10902
 
+      lokiServices:
+      - loki
```
Comment (xrstf) [inline config/monitoring/grafana/values.yaml]: Super nitpick: Can you add an empty line after this?

Comment (youssefazrak) [reply config/monitoring/grafana/values.yaml]: yup

**Review by youssefazrak — COMMENTED** [review]
Done

**Pull request merged**